nixos/modules/programs/kbdlight.nix

```diff
@@ -11,6 +11,13 @@ in
 
   config = mkIf cfg.enable {
     environment.systemPackages = [ pkgs.kbdlight ];
-    security.setuidPrograms = [ "kbdlight" ];
+
+    security.permissionsWrappers.setuid =
+    [ { program = "kbdlight";
+        source  = "${pkgs.kbdlight.out}/bin/kbdlight";
+        owner   = "root";
```

Comment:
this program, source, owner, etc, pattern repeats so often that I'm wondering whether we can hide it in a function with sensible defaults .. any ideas?